docs/tutorials/faq.md

```diff
@@ -7,7 +7,9 @@ bad content.  But now in v2, we don't.
 
 This requires setting up clock sync for live streams.  This can be done by
 adding a `<UTCTiming>` element to the manifest or by setting the
-[`.manifest.dash.clockSyncUri`][DashManifestConfiguration] configuration.
+{@link 
+https://shaka-player-demo.appspot.com/docs/api/shaka.extern.html#.DashManifestConfiguration
```

Comment:
Would you please check if {@link shaka.extern.DashManifestConfiguration | ...} works? 
Same for the below changes.
Thank you!

Reply:
No, mam, it doesn't work I checked.

Reply:
[`.manifest.dash.clockSyncUri`]{@link shaka.extern.DashManifestConfiguration} should work. Thank you!

Reply:
Yes, mam, it works should I change it everywhere.